### ISSUE TITLE
`EmptyLineSeparator`: false positive in Enum

### ISSUE THREAD

**Issue by pavelicii** [checkstyle/checkstyle#11241]
I have read check documentation: https://checkstyle.org/config_whitespace.html#EmptyLineSeparator
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
/var/tmp $ javac SomeEnum.java

/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="EmptyLineSeparator">
            <property name="allowMultipleEmptyLines" value="false"/>
            <property name="allowMultipleEmptyLinesInsideClassMembers" value="false"/>
        </module>
    </module>
</module>

/var/tmp $ cat SomeEnum.java
package some.pckg;

import java.util.function.Function;

public enum SomeEnum {

    ONE("one"),
    TWO("two");




    private final String someString;

    SomeEnum(String someString) {
        this.someString = someString;
    }

    public String getSomeString() {
        return someString;
    }
}


/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-9.2.1-all.jar -c config.xml SomeEnum.java
Checkstyle ends without errors.
```

---

Checkstyle ends without errors, but I expect violation to be found since there are more than 1 empty lines between enum constant `TWO` and filed `someString`.

**Comment by strkkk** [comment]
I could reproduce the issue, will fix it soon.

**Comment by pavelicii** [comment]
Similar issue that can be closed after the fix is applied: https://github.com/checkstyle/checkstyle/issues/10529

**Comment by rnveach** [comment]
> Checkstyle ends without errors, but I expect violation 

Doesn't this make it a false negative?

**Comment by strkkk** [comment]
This is also false negative on interface fields - https://github.com/checkstyle/checkstyle/pull/11245#discussion_r794503755
will be fixed in same pr
